File: ElasticsearchClient/src/app/shared/models/response/response-get-loggs-view.ts
export class ResponseGetLoggsViewItem {
  id: number;
  creationDate: Date | string;
  request: string;
  source: string;
  message: string;
  stackTrace: string;
  userId: string;

  constructor() {
    this.id = 0;
    this.creationDate = null;
    this.request = '';
    this.source = '';
    this.message = '';
    this.stackTrace = '';
    this.userId = '';
  }
}

export class ResponseGetLoggsView {
 items: ResponseGetLoggsViewItem[];
 count: number;

 constructor() {
  this.items = new Array<ResponseGetLoggsViewItem>();
 }
}

File: Common/Views/Loggs/Response/ResponseGetLoggsView.cs
using System;
using System.Collections.Generic;

namespace Common.Views.Loggs.Response
{
    public class ResponseGetLoggsView
    {
        public List<ResponseGetLoggsViewItem> Items { get; set; }
        public int Count { get; set; }
    }
    public class ResponseGetLoggsViewItem
    {
        public long Id { get; set; }
        public bool IsRemoved { get; set; }
        public DateTime CreationDate { get; set; }
        public string Request { get; set; }
        
        public string Source { get; set; }
        public string Message { get; set; }
        public string StackTrace { get; set; }
        public string UserId { get; set; }
    }
}
